src/routes/(app)/api/search/meilisearch.ts

```diff
@@ -0,0 +1,7 @@
+import { Meilisearch } from "meilisearch";
+import { env } from "$env/dynamic/private";
+
+export const meilisearch = new Meilisearch({
+  host: env["MEILISEARCH_HOST"],
+  apiKey: env["MEILISEARCH_MASTER_KEY"],
```

Comment:
We use dot access everywhere else as far as I can tell - maybe do the same here?